packages/vscode-extension/src/builders/buildAndroid.ts

```diff
@@ -76,14 +76,38 @@ export async function buildAndroid(
   outputChannel: OutputChannel,
   progressListener: (newProgress: number) => void
 ): Promise<AndroidBuildResult> {
+  const { buildScript, env, android } = getLaunchConfiguration();
+
+  if (buildScript) {
+    const buildProcess = cancelToken.adapt(
+      exec(buildScript.name, ["android", ...(buildScript.args ?? [])])
+    );
+    let apkPath: string | undefined;
+    lineReader(buildProcess).onLineRead((line) => {
+      apkPath = line.trim();
+    });
+
+    await buildProcess;
+
+    if (!apkPath || fs.existsSync(apkPath)) {
+      throw new Error("Build script didn't output any app path");
```

Comment:
I believe that dis error will cause webview to display buildErrorAlert, but this error alert is redirecting the user to build logs, which is not usefull in this case, could you make a special case for external builds that displays appropriate message and redirects to launchConfiguration? 

<img width="818" alt="Screenshot 2024-08-23 at 14 24 43" src="https://github.com/user-attachments/assets/54a22e49-8586-4c05-a127-1199b3a21d3f">